Aplib.Extensions/Actions/PathfinderAction.cs

```diff
@@ -0,0 +1,43 @@
+using Aplib.Core;
+using Aplib.Core.Belief.BeliefSets;
+using System;
+
+namespace Aplib.Extensions.Actions
+{
+    public class PathfinderAction<TBeliefset, TLocation> : Core.Intent.Actions.Action<TBeliefset>
+        where TBeliefset : IBeliefSet
+    {
+        public PathfinderAction(IMetadata metadata,
+            IPathfinder<TLocation> pathfinder,
+            Func<TBeliefset, TLocation> getCurrentLocation,
+            Func<TBeliefset, TLocation> getEndLocation,
```

Comment:
More clear is good clear
```suggestion
            Func<TBeliefset, TLocation> getTargetLocation,
```